docs/sources/datasources/aws-cloudwatch/_index.md

```diff
@@ -123,6 +123,12 @@ You can attach these permissions to the IAM role or IAM user you configured in [
       "Effect": "Allow",
       "Action": "tag:GetResources",
       "Resource": "*"
+    },
+    {
+      "Sid": "AmazonRDSPerformanceInsightsGetResourceMetrics",
```

Comment:
Debatably not all users need this, so we could add it separately like the cross-account observability permissions, but that felt to granular to me? Though I don't have a strong opinion on it if we want to do that instead.